odoo/addons/base/data/res_country_data.xml

```diff
@@ -698,6 +698,8 @@
             <field file="base/static/img/country_flags/hn.png" name="image" type="base64" />
             <field name="currency_id" ref="HNL" />
             <field eval="504" name="phone_code" />
+            <field eval="'%(street)s %(street2)s \n%(city)s %(state_name)s \n%(country_name)s'" name="address_format"/>
```

Comment:
Could you tell me what is different here than the standard?

Reply:
Got your point. Actually is the same as the standard format. 
I'm gonna remove that.